compose.yaml

```diff
@@ -187,7 +187,7 @@ services:
       - --host
       - http://trustify:8080
       - -u
-      - "6"
+      - "7"
```

Comment:
Why is it necessary to increase the number of users?

Reply:
Well, it will work as expected for Goose and so associating the 100 users to the transactions accordingly to the transactions' weights, i.e. 5 for `RestAPIUser` and 1 for both `RestAPIUserSlow` and `RestAPIUserDelete`.
So, there will be 14 users running the `RestAPIUserDelete` transactions which can be executed also in parallel by multiple users.